.gitignore

```diff
@@ -177,4 +177,3 @@ cython_debug/
 .ijwb
 *.db
 *.db-journal
-.DS_Store
```

Comment:
remove redundant copy of this, we have two in gitignore currently https://github.com/mozilla-ai/lumigator/blob/0ccd41f40d8fe85b1af580c546893da95b2c9cb8/.gitignore#L92